### PR TITLE
Add configuration files for an experimental setup

### PR DESCRIPTION
**Pull request by AntonioViscomi**
Added configuration files for our experimental Camozzi linear actuator setup.

### CODE REVIEW

**Review by pattacini — CHANGES_REQUESTED** [review]
Hi @AntonioViscomi ,

Could you push to your branch https://github.com/AntonioViscomi/robots-configuration/tree/ankle-setup a `README.md` file under the path `ankleSetup` providing a brief description of the context?

Once done, I'll be happy to merge the PR.

File: experimentalSetups/ankleSetup/telemetry/telemetry.xml
Comment (valegagge) [inline experimentalSetups/ankleSetup/telemetry/telemetry.xml:25]: please use a more significant name instead of `fakedevicetest`, like `ankleSetupTelemtery` or something else. Be careful to substitute the name also in the ankle-setup-rawval-nws_wrapper.xml file. thanks

File: experimentalSetups/ankleSetup/ankleSetup_telemetry_data/bufferConfigankleSetup_test.json
Comment (valegagge) [inline experimentalSetups/ankleSetup/ankleSetup_telemetry_data/bufferConfigankleSetup_test.json:1]: Please remove this file and the parent folder. thanks

**Review by valegagge — CHANGES_REQUESTED** [review]
Hi @AntonioViscomi , all is fine.I asked two minor changes only.

**Review by pattacini — CHANGES_REQUESTED** [review]
Very nice the idea of including an image 💯 
However, 1.31 MB is a bit too much for the purpose.
Could you make it smaller than 100 KB?

**Review by pattacini — CHANGES_REQUESTED** [review]
README formatting needs improvements.
Take a look at https://github.com/AntonioViscomi/robots-configuration/tree/ankle-setup/experimentalSetups/ankleSetup.

File: experimentalSetups/ankleSetup/README.md
Comment (pattacini) [inline experimentalSetups/ankleSetup/README.md:4]:
```suggestion
![](readme_assets/imgs/setup.jpg)
```

File: experimentalSetups/ankleSetup/README.md
Comment (pattacini) [inline experimentalSetups/ankleSetup/README.md:8]:
```suggestion
| `MECAPION APM SA01ACN-8` [Datasheet](http://wingst-icub.iit.local/direct_download.php?LINK=%2F%2Fiiticubstor01.iit.local%2Ficub-tech%2Fmechatronics%2FDatasheet%2FMotors%2FMecapion%2FMecapion+servo+motors+2012.pdf) 🔒 | `2FOC` [Documentation](https://github.com/icub-tech-iit/electronics-boards/tree/master/2foc/docs) |
```

We need to highlight that it's a private resource.

**Review by pattacini — APPROVED** [review]
Thank you @AntonioViscomi for the mods!
I've pointed out a few amendments as suggestions that I'll commit straight away.

**Comment by pattacini** [comment]
Awaiting @valegagge final approval before merging.

**Comment by AntonioViscomi** [comment]
Thank you @pattacini for your revision!

File: experimentalSetups/ankleSetup/ankleSetup_telemetry_data/bufferConfigankleSetup_test.json
Comment (pattacini) [reply experimentalSetups/ankleSetup/ankleSetup_telemetry_data/bufferConfigankleSetup_test.json]: Done!

File: experimentalSetups/ankleSetup/telemetry/telemetry.xml
Comment (pattacini) [reply experimentalSetups/ankleSetup/telemetry/telemetry.xml]: Done!

**Comment by pattacini** [comment]
I've seen that you addressed @valegagge's concerns so I think we can make it within 2024!
Feel free to drop further comments.

**Pull request merged**
